src/org/openj9/envInfo/EnvDetector.java

```diff
@@ -72,6 +74,7 @@ private static void getJavaInfo() {
 			output.write(JDKVERSIONvalue);
 			output.write(JDKIMPLvalue);
 			output.write(JDKVENDORvalue);
+			output.write(ReleaseInfo); // TOOD: Add to "AQACert.log" file when rebase
```

Comment:
This cannot be output into autoGenEnv.mk. This info should be AQACert.log.
We should wait and get PR https://github.com/adoptium/TKG/pull/266/ completed first.

Reply:
Would like to check the workflow here as it's my first time working in a forked repo.

My thoughts are:
1. Cherrypick the merged `260_release change` that's merged later yesterday
2. Rease on the `260_release change`

Please let me know if I'm mistaken, thanks!

Reply:
Used github client to resolve